Soccar [Game-Server]/src/nl/soccar/gameserver/util/MapUtilities.java

```diff
@@ -27,7 +27,7 @@ public static void addWalls(GameEngine engine, Map map) {
         addWestWalls(engine, map, mapHeight);
         addEastWalls(engine, map, mapWidth, mapHeight);
         addNorthAndSouthWalls(engine, mapWidth, mapHeight);
-        addCornerWalls(engine, mapWidth, mapHeight);
+        //addCornerWalls(engine, mapWidth, mapHeight);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)